DESCRIPTION

```diff
@@ -29,15 +29,16 @@ Imports:
     lifecycle,
     magrittr,
     purrr (>= 0.2.3),
-    rlang (>= 0.3.1),
+    rlang (>= 0.4.11.9000),
     tibble (>= 2.0.0),
     utils,
     uuid,
+    vctrs,
```

Comment:
Should I be putting a minimum version on vctrs?

Reply:
I would use at least 0.3.0, as that was the version where we changed the ptype2/cast system to what you are using here